ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/CapellaExecutionClientHandler.java

```diff
@@ -36,15 +39,38 @@
 public class CapellaExecutionClientHandler extends BellatrixExecutionClientHandler
     implements ExecutionClientHandler {
   private static final Logger LOG = LogManager.getLogger();
+  private Optional<UInt64> firstCapellaSlot = Optional.empty();
```

Comment:
I think we could reduce the noise by making this final and non-optional, like:
<img width="954" alt="image" src="https://user-images.githubusercontent.com/15999009/212121089-72305688-8eef-4b33-acfb-c4fb7e01ea5e.png">

We shouldn't try to instantiate this handler if Capella is not even defined..